src/components/TopNavigation/TopNavigation.tsx

```diff
@@ -8,11 +8,11 @@ const TopNavigation = () => {
   const userName: string = "John Doe";
   return (
     <>
-      <div className="relative">
-        <div className="absolute top-3 left-3">
+      <div className="flex">
```

Comment:
nice that u changed to flex, but i think fixed should be used only here for whole nav and elements inside should be positioned using flex with gaps and paddings, not top left etc
```suggestion
      <div className="flex fixed">
```